demo/default.js

```diff
@@ -208,7 +206,7 @@
   // Update Markdown from hash (if present)
   if (window.location.hash) {
     try {
-      const decodedHash = decodeURIComponent(window.location.hash.substring(1));
+      var decodedHash = decodeURIComponent(window.location.hash.substring(1));
```

Comment:
This got flagged while testing the ES3/ES5 parseOption, but `const` wasn't used elsewhere in the file so I kept that change here

Reply:
Ooh, nice!